variables.tf

```diff
@@ -328,21 +328,24 @@ variable "repositories" {
     }
 
     mageos-async-events-aws = {
-      description = "This module provides events integrations to AWS."
-      teams       = ["distribution", "async-events"]
-      topics      = ["mage-os", "magento", "ecommerce", "magento2", "adobecommerce", "aws"]
+      description     = "This module provides events integrations to AWS."
+      teams           = ["distribution", "async-events"]
+      topics          = ["mage-os", "magento", "ecommerce", "magento2", "adobecommerce", "aws"]
+      mono_repository = true
```

Comment:
Does this flag mean that it's part of a mono repo? Or that it's a mono repo itself? Can you disambiguate the flag?

Reply:
It means that it is part of a mono repo. Do you have an idea for a better name? I currently cannot think of any.

Reply:
Maybe one of these?
* `is_part_of_monorepo`
* `belongs_to_monorepo`
* `monorepo_part`
* `split_monorepo_part`
* `monorepo_slice`

Reply:
- `monorepo_part`
- `monorepo_component`
- `is_part_of_monorepo`
- `belongs_to_monorepo` 

are my favorite.